terraform_alert_config/import_adapter/README.md

```diff
@@ -0,0 +1,68 @@
+# Prepare workspace
```

Comment:
Let's add a summary section to the top:

`
## Terraform Alerting Config

This document describes the example DICOM Import adapter terraform configuration. The terraform configuration deploys a DICOM Import adapter to Google Kubernetes Engine, creates a Cloud Healthcare dataset and dicom store, and configures monitoring to alert on the health of the adapter.
`